andromeda-ui/backend/observations.ts

```diff
@@ -10,12 +11,21 @@ export async function fetchObservations(iNatUser: string, addSatRGB: boolean, ad
     });
 }
 
-export function makeObservationURL(iNatUser: string, addSatRGB: boolean, addLandCover: boolean, format: "json" | "csv") {
-  let base_url = "/inaturalist/" + iNatUser;
-  if (format) {
-    base_url += "?format=" + format;
+export function makeObservationURL(iNatUser: string, addLandCover: boolean, 
+      format: "json" | "csv", maxObs?: number) {
+  let base_url = "/inaturalist/" + iNatUser + "?format=" + format;
+  if (maxObs) {
+    base_url += "&limit=" + maxObs;
   }
-  base_url += "&add_sat_rgb_data=" + addSatRGB;
   base_url += "&add_landcover_data=" + addLandCover;
   return apiURL(base_url);
 }
+
+export function downloadSecondsEstimate(numObservations: number) {
+  const seconds = numObservations * 5;
```

Comment:
Presumably `5` is the time per request? Perhaps make that a constant somewhere so it's easy to find and change later if they scale up their infrastructure?

Reply:
Good suggestion. I pushed a commit to better document this constant.

Reply:
Maybe we can make an issue to add a Workflow to auto-test the iNat API once a week or once a month to update the latency per request in a config file that this download estimation function can read from to make a relatively up-to-date estimate?

Reply:
That sounds like a really big project for little gain. My goal with these estimates is just to get the user to know they need to wait for a while. If you feel strongly about this please create an issue for this.